appengine/Rakefile

```diff
@@ -0,0 +1,19 @@
+$VERBOSE = true
+
+task :default => [:test]
+
+task :build do
+  ::Dir.chdir(::File.dirname(__FILE__)) do
+    puts("docker build -t appengine-ruby-base .")
+    unless system("docker build -t appengine-ruby-base .")
+      raise "Exit code #{$?} when running docker build"
+    end
+  end
+end
+
+task :test => [:build] do
```

Comment:
Lets use Rake::TestTask, as this breaks things if you provide more tasks to do after test.